src/main/java/org/launchcode/techjobs/persistent/models/Job.java

```diff
@@ -1,51 +1,63 @@
 package org.launchcode.techjobs.persistent.models;
 
+
 import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
 
 @Entity
-public class Job{
+public class Job extends AbstractEntity{
+
+    @ManyToOne
+    private Employer employer;
+
+    @ManyToMany
+    private List<Skill> skills = new ArrayList<>();
 
-    @Id
-    @GeneratedValue
-    private int id;
+    //@Id
```

Comment:
This is no problem throughout the class. But try to only leave comments to explain how your code works. Having too many comments in code can make it confusing. Just something to think about in the future :)